src/ansi.rs

```diff
@@ -1551,6 +1554,28 @@ where
 
                 handler.clear_line(mode);
             },
+            ('k', [b' ']) => {
+                // SCP control
+                //
+                // Changes from ECMA-48 as proposed by the BiDi draft proposal:
+                //  * Recognize value 0 for Ps1
+                //  * Allow passing no params or Ps1 alone
```

Comment:
Is this actually used by applications? I don't really mind it but I don't see an advantage in applications supporting a newer "spec" that is much more likely to cause only partial support.